.github/workflows/on_merge_to_main.yml

```diff
@@ -136,10 +136,10 @@ jobs:
           # Chromatic needs the full git history to work
           fetch-depth: 0
 
-      - name: Setup Node.js 12.x
+      - name: Setup Node.js 16.13
```

Comment:
question: why do we sometimes add the suffix `.x` and sometimes not?

Reply:
When I'm using it in a name without the `.x`, I'm just being lazy. 

When I'm using it in the actual node version, I'm pinning Node to a specific minor version, not just a major version. It might work with out the `.x` but 🤷🏻‍♂️